main/store/migrations/index.js

```diff
@@ -481,6 +481,15 @@ const migrations = {
       chain.isTestnet = chain.layer === 'testnet'
     })
 
+    return initial
+  },
+  23: (initial) => {
+    // set icon and primaryColor values on all chains
+    Object.values(initial.main.networksMeta.ethereum).forEach(chain => {
+      chain.icon = ''
```

Comment:
can we update the NetworkMetadata type in the state.d.ts definition file?

Reply:
[floating/frame@`ce3baff` (#1066)](https://github.com/floating/frame/pull/1066/commits/ce3bafff2c264de7264246446bb8149c5d911426)